src/server.js

```diff
@@ -0,0 +1,33 @@
+const app = require('./app');
+const http = require('http');
+const mongoose = require('mongoose');
+
+const uri = ('mongodb://localhost:27017/test' || process.env.mongoSecret);
+mongoose.connect(uri, {useNewUrlParser: true, useFindAndModify: false})
+.then(() => {
```

Comment:
This is a no-op, and can be removed

Reply:
Added a console.log() for now. Can remove if unnecessary.